spike_main/spike_device.cc

```diff
@@ -298,7 +298,7 @@ int spike_device::run(meta_data* knl_data,uint64_t knl_start_pc){
   uint64_t knlbase=knl_data->metaDataBaseAddr;
 
   if ((ldssize)>0x10000000) {
-        fprintf(stderr, "lds size is too large. please modify VBASEADDR");
+        DEBUG(fprintf(stderr, "lds size is too large. please modify VBASEADDR");)
```

Comment:
Indent

Reply:
> Indent

The program's coding style maybe later needed to be modified,I think the maintainer is supposed to create a PR to do this, I will later modify what I do in this PR